Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +11,33 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if(array != null && array.Length != 0)
+            {
+                var arrayLength = array.Length;
+                // loop through all elements
+                for (var i = 0; i < arrayLength; i++)
+                {
+                    // compare element on current index to all next elements
+                    for (var j = i + 1; j < arrayLength; j++)
+                    {
+                        // keep swapping till lowest is found
+                        if (array[i] > array[j])
+                        {
+                            var tmp = array[i];
+                            array[i] = array[j];
+                            array[j] = tmp;
+
+                            //Console.WriteLine("Swapping: " + array[j] + " and " + array[i]);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5011611)